src/patternfly/demos/Toolbar/examples/toolbar-attribute-value-search-filter-example.hbs

```diff
@@ -0,0 +1,79 @@
+{{#> data-toolbar data-toolbar--id="data-toolbar-attribute-value-search-filter-example"}}
+  {{#> data-toolbar-content}}
+    {{#> data-toolbar-group data-toolbar-group--modifier="pf-m-toggle-group pf-m-reveal-on-lg" data-toolbar-group--IsExpanded="false"}}
+      {{> data-toolbar-toggle}}
+      {{#> data-toolbar-group data-toolbar-group--modifier="pf-m-filter-group"}}
+      {{#> data-toolbar-item}}
+        {{#> select id="select-two" select-toggle--icon="fas fa-filter"}}
+          Name
+        {{/select}}
+      {{/data-toolbar-item}}
+      {{#> data-toolbar-item}}
+        {{#> input-group}}
+          {{#> form-control controlType="input" input="true" form-control--attribute='type="search" id="textInput11" name="textInput11" aria-label="search input example"'}}
```

Comment:
Same with creating the `id` for this using `data-toolbar--id`, even though it's only in this demo